docs/converters/stringtolistconverter.md

```diff
@@ -0,0 +1,81 @@
+---
+title: "Xamarin Community Toolkit StringToListConverter"
+author: guoldev
+ms.author: joverslu
+description: "The StringToListConverter allows the users to retrieve convert a string value into a string array that contains the substrings in this string that are delimited by Separator value or Separators."
+ms.date: 01/04/2021
+---
+
+# Xamarin Community Toolkit StringToListConverter
+
+The StringToListConverter is a converter that allows the users to retrieve convert a `string` value into a string array that contains the substrings in this string that are delimited by the `Separator` or `Separators` or `ConverterParameter`.
+
+## Syntax
+
+```
+<?xml version="1.0" encoding="UTF-8" ?>
+<pages:BasePage
+    x:Class="Xamarin.CommunityToolkit.Sample.Pages.Converters.StringToListConverterPage"
+    xmlns="http://xamarin.com/schemas/2014/forms"
+    xmlns:x="http://schemas.microsoft.com/winfx/2009/xaml"
+    xmlns:pages="clr-namespace:Xamarin.CommunityToolkit.Sample.Pages"
+    xmlns:xct="http://xamarin.com/schemas/2020/toolkit">
+    <pages:BasePage.Resources>
+        <ResourceDictionary>
+            <xct:StringToListConverter x:Key="StringToListConverter" SplitOptions="RemoveEmptyEntries">
+                <xct:StringToListConverter.Separators>
+                    <x:String>,</x:String>
+                    <x:String>.</x:String>
+                    <x:String>;</x:String>
+                </xct:StringToListConverter.Separators>
+            </xct:StringToListConverter>
+        </ResourceDictionary>
+    </pages:BasePage.Resources>
+    <pages:BasePage.Content>
+        <Grid Margin="20,0" RowDefinitions="Auto,Auto,*">
+            <Label
+                Grid.Row="0"
+                FontAttributes="Bold"
+                Text="Enter some text separated by ',' or '.' or ';'" />
+            <Entry
+                x:Name="ExampleText"
+                Grid.Row="1"
+                FontSize="Medium"
+                Placeholder="Enter some text separated by ',' or '.' or ';'"
+                Text="Item 1,Item 2,Item 3" />
+            <ListView
+                Grid.Row="2"
+                BindingContext="{x:Reference Name=ExampleText}"
+                ItemsSource="{Binding Path=Text, Converter={StaticResource StringToListConverter}}">
+                <ListView.ItemTemplate>
+                    <DataTemplate>
+                        <ViewCell>
+                            <Label FontSize="Medium" Text="{Binding .}" />
+                        </ViewCell>
+                    </DataTemplate>
+                </ListView.ItemTemplate>
+            </ListView>
+        </Grid>
+    </pages:BasePage.Content>
+</pages:BasePage>
+```
+
+## Properties
+
+<!-- Explain all properties in a table format -->
+
+| Property | Type | Description |
+| -- | -- | -- |
+| Separator | string | The string that delimits the substrings in this string. |
+| Separators | string[] | The strings that delimits the substrings in this string. |
```

Comment:
The `Type` shouldn't be `IList<string>` ?

Reply:
@pictos Thanks for the review, I have fixed it, but the `<string>` is not displayed if you open the [file](https://github.com/GUOLDEV/xamarin-communitytoolkit/blob/stringtolistconverter-doc/docs/converters/stringtolistconverter.md). Do you know how to fix it?